nix/fileFilter.nix

```diff
@@ -37,6 +37,8 @@ let
     ../Cargo.toml
     ../Cargo.lock
     ../version.txt
+    ../rust-toolchain.toml
+    ../.cargo/config.toml
```

Comment:
config.toml was necessary for the debug build and including rust-toolchain.toml seems smart as well.